Samples/iOS-Swift/iOS-Swift/ViewControllers/TraceTestViewController.swift

```diff
@@ -22,13 +22,16 @@ class TraceTestViewController: UIViewController {
         }
         let session = URLSession(configuration: URLSessionConfiguration.default)
         let dataTask = session.dataTask(with: imgUrl) { (data, _, error) in
+            //Simulated delay in the download
+            Thread.sleep(forTimeInterval: 2)
```

Comment:
If you mean UI testing, we dont have a test for this screen. 
But  if you mean manual test, I guess we also need a way to test `reportFullyDisplayed`, and this would be the place.

Reply:
I wonder why we need the thread sleep. We already load some data via an HTTP request.

Reply:
The load was so fast, maybe cache involved, that I couldn't properly test time to full display